packages/SymbioticController/src/Adapter/Nette/Application/InvokablePresenterAwareApplication.php

```diff
@@ -154,21 +155,21 @@ private function dispatchException(Throwable $exception): void
     private function dispatchApplicationResponseEvent(): void
     {
         $this->eventDispatcher->dispatch(
-            PresenterCreatedEvent::class, new PresenterCreatedEvent($this, $this->presenter)
+            ApplicationEvents::ON_PRESENTER, new CallablePresenterEvent($this, $this->presenter)
         );
     }
 
     private function dispatchApplicationException(Throwable $exception): void
     {
         $this->eventDispatcher->dispatch(
-            ApplicationErrorEvent::class, new ApplicationErrorEvent($this, $exception)
+            ApplicationEvents::ON_ERROR, new ErrorEvent($this, $exception)
         );
     }
 
     private function dispatchApplicationShutdownException(Throwable $exception): void
     {
         $this->eventDispatcher->dispatch(
-            ApplicationShutdownEvent::class, new ApplicationShutdownEvent($this, $exception)
+            ApplicationEvents::ON_ERROR, new ErrorEvent($this, $exception)
```

Comment:
Is this correct? `ApplicationShutdownEvent` vs `ErrorEvent` at least the naming seems to be weird to me.